### ISSUE TITLE
Any mention or just "thank you" for Tipsi

### ISSUE THREAD

**Issue by isnifer** [expo/expo#521]
Hi, guys! Since you are using `tipsi-stripe` component internally in expo, I would like to see any mention of our team.
Also I have one question – how are you planning support this code in paralllel with `tipsi-stripe`?

**Comment by brentvatne** [comment]
good point @isnifer! ping @ide @dikaiosune @jeff-da

**Comment by brentvatne** [comment]
@jeff-da pointed out that we do credit tipsi-stripe in the docs:
 
<img width="912" alt="screen shot 2017-08-19 at 5 09 47 pm" src="https://user-images.githubusercontent.com/90494/29490947-3a268ab4-8501-11e7-8fc0-f02d50e144a7.png">
https://docs.expo.io/versions/latest/sdk/payments.html

we could make it more prominent if you like :) also, thank you @isnifer and tipsi!

> Also I have one question – how are you planning support this code in paralllel with tipsi-stripe?

could you elaborate on this? I'm not sure I understand the question.

**Comment by isnifer** [comment]
@brentvatne We have a lot of issues, plans how to make `tipsi-stripe` better. Will you just grab code into Expo or develop it independently?

**Comment by isnifer** [comment]
@brentvatne and just one more, if it possible while you here, please, take a look - https://github.com/expo/ex-navigation/pull/492

**Comment by dikaiosune** [comment]
@isnifer we update the libraries we vendor regularly. Definitely open an issue if a critical update has released that we've missed!

Also, I bumped the credit for tipsi to the top of our docs: https://github.com/expo/expo-docs/commit/dfbfd43c253af4e175fe841d87ad72008d1ec7f1

Looks like that ex-navigation PR was merged.

Think that addresses all of the questions/concerns here, giving it a close. Thanks again for working on tipsi!!!
